### PR TITLE
feat: Initial logic to disable all but core mods

### PR DESCRIPTION
**Pull request by GeckoEidechse**
In the future, this button should be moved to a "Repair" view. For now I'm putting it into Dev view for testing.

### CODE REVIEW

File: src-vue/src/views/DeveloperView.vue [hunk 1]
```diff
@@ -53,6 +61,30 @@ export default defineComponent({
...
                 position: 'bottom-right'
             });
+        },
+        async disableAllModsButCore() {
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+            await invoke("disable_all_but_core_caller", { gameInstall: game_install }).then((message) => {
+                console.log(message);
```
Comment (Alystrasz) [inline src-vue/src/views/DeveloperView.vue:71]: This should be removed.

File: src-vue/src/views/DeveloperView.vue [hunk 2]
```diff
@@ -53,6 +61,30 @@ export default defineComponent({
...
+                ElNotification({
+                    title: 'Success',
+                    message: "Disabled all mods but core",
+                    type: 'success',
+                    position: 'bottom-right'
+                });
+            })
+                .catch((error) => {
+                    console.error(error);
```
Comment (Alystrasz) [inline src-vue/src/views/DeveloperView.vue:80]: This should be removed.

File: src-vue/src/views/DeveloperView.vue [hunk 3]
```diff
@@ -53,6 +61,30 @@ export default defineComponent({
...
+                    ElNotification({
+                        title: 'Error',
+                        message: `${error}`,
```
Comment (Alystrasz) [inline src-vue/src/views/DeveloperView.vue:83]: Why not `message: error`?

**Review by Alystrasz — CHANGES_REQUESTED** [review]

File: src-vue/src/views/DeveloperView.vue (hunk 2 above)
Comment (GeckoEidechse) [reply src-vue/src/views/DeveloperView.vue]: Done with 3359b95eb3defa968ee391bc9ca7426754e5e78f

File: src-vue/src/views/DeveloperView.vue (hunk 1 above)
Comment (GeckoEidechse) [reply src-vue/src/views/DeveloperView.vue]: Done with 3359b95eb3defa968ee391bc9ca7426754e5e78f

**Pull request merged**